### PR TITLE
Move swift-bindings into libx10.so

### PR DESCRIPTION
**Pull request by pschuh**

### CODE REVIEW

File: Sources/x10/CMakeLists.txt [hunk 1]
```diff
@@ -1,18 +1,9 @@
...
-set_target_properties(x10_c_wrappers PROPERTIES
-  POSITION_INDEPENDENT_CODE ON)
-target_link_libraries(x10_c_wrappers PUBLIC
-  x10)
-
 add_library(x10_device SHARED
   swift_bindings/Device.swift)
-target_link_libraries(x10_device PRIVATE
-  x10_c_wrappers)
+target_link_libraries(x10_device PUBLIC
```
Comment (compnerd) [inline Sources/x10/CMakeLists.txt:5]: I think that the `PUBLIC` should be `PRIVATE`.

File: Sources/x10/swift_bindings/BUILD [hunk 2]
```diff
@@ -0,0 +1,33 @@
+cc_library(
+    name = "device_wrapper",
+    srcs = ["device_wrapper.cc"],
+    hdrs = ["device_wrapper.h"],
+    deps = [
+        "//tensorflow/compiler/xla/xla_client:xrt_computation_client",
```
Comment (compnerd) [inline Sources/x10/swift_bindings/BUILD:6]: Indentation is odd

File: Sources/x10/swift_bindings/BUILD [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    ],
+)
+
+cc_library(
+    name = "xla_tensor_wrapper",
+    srcs = ["xla_tensor_wrapper.cc"],
+    hdrs = ["xla_tensor_wrapper.h"],
+    deps = [
+        ":device_wrapper",
+	"//tensorflow/compiler/tf2xla/xla_tensor:tensor",
```
Comment (compnerd) [inline Sources/x10/swift_bindings/BUILD:17]: Indentation is odd

File: Sources/x10/swift_bindings/BUILD [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    ],
+)
+
+cc_library(
+    name = "xla_tensor_tf_ops",
+    srcs = ["xla_tensor_tf_ops.cc"],
+    hdrs = ["xla_tensor_tf_ops.h"],
+    deps = [
+        ":xla_tensor_wrapper",
+	"//tensorflow/compiler/tf2xla/xla_tensor:tensor",
```
Comment (compnerd) [inline Sources/x10/swift_bindings/BUILD:29]: Indentation is odd.

**Comment by pschuh** [comment]
> Should we remove the `x10_c_wrappers` target from the cmake script then?

done.

**Comment by compnerd** [comment]
@googlebot: rescane

**Pull request merged**